### PR TITLE
feat/fix(legacy): updated mojang -> minecraftservices api & fixed session login

### PR DESCRIPTION
**Pull request by EclipsesDev**
also fixed glow storageesp/esp color

Closes: #2243

### CODE REVIEW

File: src/main/java/net/ccbluex/liquidbounce/features/module/modules/render/StorageESP.kt [hunk 1]
```diff
@@ -303,7 +302,5 @@ object StorageESP : Module("StorageESP", Category.RENDER) {
         } catch (ex: Exception) {
             LOGGER.error("An error occurred while rendering all storages for shader esp", ex)
         }
-
-       GlowShader.stopDraw(Color(0, 66, 255), glowRadius, glowFade, glowTargetAlpha)
```
Comment (DataM0del) [inline src/main/java/net/ccbluex/liquidbounce/features/module/modules/render/StorageESP.kt]: Here?

File: src/main/java/net/ccbluex/liquidbounce/features/module/modules/render/StorageESP.kt [hunk 2]
```diff
@@ -263,7 +263,6 @@ object StorageESP : Module("StorageESP", Category.RENDER) {
             return
 
         val renderManager = mc.renderManager
-        GlowShader.startDraw(event.partialTicks, glowRenderScale)
```
Comment (DataM0del) [inline src/main/java/net/ccbluex/liquidbounce/features/module/modules/render/StorageESP.kt:284]: This

File: src/main/java/net/ccbluex/liquidbounce/features/module/modules/render/ESP.kt [hunk 3]
```diff
@@ -225,8 +223,6 @@ object ESP : Module("ESP", Category.RENDER, hideModule = false) {
         }
 
         renderNameTags = true
-
-        GlowShader.stopDraw(getColor(), glowRadius, glowFade, glowTargetAlpha)
```
Comment (DataM0del) [inline src/main/java/net/ccbluex/liquidbounce/features/module/modules/render/ESP.kt:232]: Is

File: src/main/java/net/ccbluex/liquidbounce/features/module/modules/render/ESP.kt [hunk 4]
```diff
@@ -204,8 +204,6 @@ object ESP : Module("ESP", Category.RENDER, hideModule = false) {
         if (mc.theWorld == null || mode != "Glow")
             return
 
-        GlowShader.startDraw(event.partialTicks, glowRenderScale)
-
```
Comment (DataM0del) [inline src/main/java/net/ccbluex/liquidbounce/features/module/modules/render/ESP.kt:211]: Why

File: src/main/java/net/ccbluex/liquidbounce/utils/login/UserUtils.kt [hunk 5]
```diff
@@ -32,64 +45,71 @@ object UserUtils {
...
+    fun getUUID(username: String): String {
+        usernameCache[username]?.let { return it }
+
+        return try {
+            val url = URL("https://api.minecraftservices.com/minecraft/profile/lookup/name/$username")
+            (url.openConnection() as HttpsURLConnection).apply {
+                connectTimeout = 2000
+                readTimeout = 2000
+                requestMethod = "GET"
+                setRequestProperty("User-Agent", "Mozilla/5.0 (Windows NT 6.1; WOW64; rv:25.0) Gecko/20100101 Firefox/25.0")
```
Comment (DataM0del) [inline src/main/java/net/ccbluex/liquidbounce/utils/login/UserUtils.kt]: NT 6.1 :skull:
Windows 7...
Really old, but whatever.

**Review by DataM0del — APPROVED** [review]
Why is there random changes everywhere lol

**Comment by DataM0del** [comment]
> also fixed glow storageesp/esp color

I think you should have that as a different PR though but also Ok.

File: src/main/java/net/ccbluex/liquidbounce/features/module/modules/render/ESP.kt (hunk 4 above)
Comment (EclipsesDev) [reply src/main/java/net/ccbluex/liquidbounce/features/module/modules/render/ESP.kt:211]: duplicate

File: src/main/java/net/ccbluex/liquidbounce/utils/login/UserUtils.kt (hunk 5 above)
Comment (EclipsesDev) [reply src/main/java/net/ccbluex/liquidbounce/utils/login/UserUtils.kt]: I will update it don't worry

**Pull request merged**